### PR TITLE
Support cross-region copying in DocDB and Neptune

### PR DESCRIPTION
**Pull request by trivikr**
Verified that tests are successful:
* Unit: `./node_modules/.bin/mocha test/services/rds.spec.js`
* Integration: `./node_modules/.bin/cucumberjs --tag @rds`

##### Checklist

- [x] `npm run test` passes
- [x] changelog is added, `npm run add-change`
- [x] run `npm run integration` if integration test is changed

### CODE REVIEW

**Review by AllanZhengYP — COMMENTED** [review]
Looks good! Can you add test for Neptune and DocDB similar to RDS tests?

File: clients/docdb.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ var apiLoader = AWS.apiLoader;
 
 apiLoader.services['docdb'] = {};
 AWS.DocDB = Service.defineService('docdb', ['2014-10-31']);
+require('../lib/services/docdb');
+
```
Comment (AllanZhengYP) [inline clients/docdb.js:9]: This empty line seems will be removed in the next time client is regenerated.

**Comment by trivikr** [comment]
The unit tests are failing as SourceRegion keu is not present in rules.input.params for DocDB and Neptune operations https://github.com/aws/aws-sdk-js/blob/d810e4ea90eec60c9b053e8ae4786bda806fa52f/lib/service.js#L193-L197

**Comment by trivikr** [comment]
Added support for un-modeled SourceRegion parameter as per spec in [`1e7a734` (#3628)](https://github.com/aws/aws-sdk-js/pull/3628/commits/1e7a734239089438109b4224866dea27afd7f0cd)

**Review by AllanZhengYP — APPROVED** [review]

**Pull request merged**